### PR TITLE
Yaz 32/solve endpoint errors

### PR DESCRIPTION
**Pull request by juanDuqueS**
What: solve errors in the endpoint

why: To have a functional conection

Yaz-32/solve endpoint errors

### CODE REVIEW

File: back-end-yaz/src/server/controllers/UserController.js [hunk 1]
```diff
@@ -58,7 +58,7 @@ class UserController {
             if (!Number(id)) {
                 util.setError(400, 'Please input a valid numeric value');
             } else if(theUser){
-                util.setSuccess(200, `User ${id} returned!`);
+                util.setSuccess(200, `User ${id} returned!`, theUser);
```
Comment (AngelicaAgudelo) [inline back-end-yaz/src/server/controllers/UserController.js:61]: Please changes this var name

File: back-end-yaz/src/server/controllers/ServiceController.js [hunk 2]
```diff
@@ -58,7 +58,7 @@ class ServiceController {
             if (!Number(id)) {
                 util.setError(400, 'Please input a valid numeric value');
             } else if(theService){
-                util.setSuccess(200, `Service ${id} returned!`);
+                util.setSuccess(200, `Service ${id} returned!`, theService);
```
Comment (AngelicaAgudelo) [inline back-end-yaz/src/server/controllers/ServiceController.js:61]: Please changes this var name

Comment (juanDuqueS) [reply back-end-yaz/src/server/controllers/ServiceController.js]: I will changed it in just a second

**Review by AngelicaAgudelo — APPROVED** [review]
Ok.

**Review by alejandroVaron — APPROVED** [review]
Nice work!

**Pull request merged**